src/state/recordedState.ts

```diff
@@ -56,7 +56,7 @@ export class RecordedState {
       this.commandList.length > 0
     ) {
       // Used for the registers and macros that only record on commandList
```

Comment:
Let's clarify & expand this comment - it's not obvious why we're doing this early return.

Reply:
The commandList here saves the ex command, which typically do not include leader. 

https://github.com/VSCodeVim/Vim/blob/5365305330a69288f38acfd12ae85cba7a593fa6/src/state/recordedState.ts#L58-L59

Therefore, we do not need the following replace

https://github.com/VSCodeVim/Vim/blob/5365305330a69288f38acfd12ae85cba7a593fa6/src/state/recordedState.ts#L62-L64

Reply:
I think J-Fields means your explanation should be in the comment in the code rather than in the comment on this PR.

Reply:
Oh, I understand. Let me add a comment